pogobuf/pogobuf.client.js

```diff
@@ -1107,7 +1108,17 @@ function Client(options) {
                             api_url: responseEnvelope.api_url
                         });
 
-                        resolve(self.callRPC(requests, envelope));
+                        signedEnvelope.platform_requests = [];
+                        resolve(self.callRPC(requests, signedEnvelope));
+                        return;
+                    }
+
+                    /* Throttling, retry same request later */
+                    if (responseEnvelope.status_code === 52) {
+                        signedEnvelope.platform_requests = [];
+                        Promise.delay(2000).then(() => {
```

Comment:
That part is fine, I was just wondering if there is a better way to handle this than to implement a fixed 2000 ms delay and retry. That's why I thought we might be able to use `bluebird-retry` like we already do elsewhere with a configurable delay and maximum number of tries.